<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="check">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>760</y>
      <width>480</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #0078D7;
    color: white;
    font-size: 18px;
    font-weight: bold;
    border: none;
    border-radius: 15px;
    padding: 10px;
    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.3);
}

QPushButton:hover {
    background-color: #005A9E; 
    cursor: pointer;
}

QPushButton:pressed {
    background-color: #004578;  pressed effect */
    box-shadow: 0px 2px 3px rgba(0, 0, 0, 0.3);
    transform: scale(0.98); 
}
</string>
    </property>
    <property name="text">
     <string>Explore Popular Movies</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>110</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_4">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>110</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_5">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>110</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>510</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_8">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>510</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_9">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>510</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel_10">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>510</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-10</y>
      <width>1921</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="navbarLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}


</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}


</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>450</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}


</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>450</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}



</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>450</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}



</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>850</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}


</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>850</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}


</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>935</x>
      <y>850</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}



</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>1315</x>
      <y>850</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}



</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>850</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: #32CD32;
color: black;
border-radius: 5px;
}


QPushButton:hover {
    background-color: #006400;
    cursor: pointer;
}



</string>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check</sender>
   <signal>clicked()</signal>
   <receiver>check</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>959</x>
     <y>919</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>919</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
